### PR TITLE
Corrupt picture error controlled

### PR DESCRIPTION
**Pull request by fesave**
## Related Issues
App: #3441 

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml [hunk 1]
```diff
@@ -16,51 +16,88 @@
...
+        android:id="@+id/error_image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:contentDescription="@string/downloader_download_failed_ticker"
+        android:src="@drawable/image_fail"
+        app:layout_constraintBottom_toBottomOf="@id/horizontal_guideline_50_percent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="@id/horizontal_guideline_50_percent" />
```
Comment (abelgardep) [inline owncloudApp/src/main/res/layout/preview_image_fragment.xml:58]: If we constrain it bottom and top to the `horizontal_guideline_50_percent`. Why couldn't we constrain it to parent?

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml [hunk 2]
```diff
@@ -16,51 +16,88 @@
...
-    tools:context=".ui.preview.PreviewImageFragment"
     android:filterTouchesWhenObscured="true"
-    >
+    tools:context=".ui.preview.PreviewImageFragment">
 
     <ProgressBar
         android:id="@+id/progressWheel"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
```
Comment (abelgardep) [inline owncloudApp/src/main/res/layout/preview_image_fragment.xml:33]: Some attributes were only useful in RelativeLayout. If we update it to constraint, we could remove some of them. For example, this one. Don't u think?

Comment (abelgardep) [inline owncloudApp/src/main/res/layout/preview_image_fragment.xml:33]: Some attributes were only useful in RelativeLayout. If we update it to constraint, we could remove some of them. For example, this one. Don't u think?

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml [hunk 3]
```diff
@@ -16,51 +16,88 @@
...
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="@id/horizontal_guideline_50_percent" />
+
+    <TextView
+        android:id="@+id/errorText"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="40dp"
+        android:gravity="center"
+        android:text="@string/downloader_download_failed_ticker"
```
Comment (abelgardep) [inline owncloudApp/src/main/res/layout/preview_image_fragment.xml:66]: I would update the message to make it generic. If the file is corrupted and downloaded, the download didn't fail. We failed to display it to the user. Make it generic till a better solution arrives 👍

Comment (abelgardep) [inline owncloudApp/src/main/res/layout/preview_image_fragment.xml:66]: I would update the message to make it generic. If the file is corrupted and downloaded, the download didn't fail. We failed to display it to the user. Make it generic till a better solution arrives 👍

**Review by abelgardep — CHANGES_REQUESTED** [review]
Some suggestions here @fesave

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml (hunk 1 above)
Comment (fesave) [reply owncloudApp/src/main/res/layout/preview_image_fragment.xml:58]: I think I did the dumbest thing in the world, I like to use guidelines so much, that sometimes I forget the basics, good catch hahaha

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml (hunk 2 above)
Comment (fesave) [reply owncloudApp/src/main/res/layout/preview_image_fragment.xml:33]: You're right. Done

File: owncloudApp/src/main/res/layout/preview_image_fragment.xml (hunk 3 above)
Comment (fesave) [reply owncloudApp/src/main/res/layout/preview_image_fragment.xml:66]: Done

**Review by abelgardep — APPROVED** [review]
First step 👍  LGTM

**Comment by jesmrec** [comment]
Approved

**Pull request merged**